.asf.yaml

```diff
@@ -48,9 +48,9 @@ github:
         required_approving_review_count: 2
 
 notifications:
-  pullrequests_status: dev@hugegraph.apache.org
+  #pullrequests_status: dev@hugegraph.apache.org
   # before use the config, we should ensure the "mail" address has set well (exist)
   #pullrequests_comment: issues@hugegraph.apache.org
   #issues: issues@hugegraph.apache.org
-  #discussions: issues@hugegraph.apache.org
+  discussions: dev@hugegraph.apache.org
```

Comment:
can add a comment to explain discussions?

Reply:
> can add a comment to explain discussions?

maybe add a link is enough?
![image](https://user-images.githubusercontent.com/17706099/221415454-5df6d49b-5a13-4f6e-9e4b-f9211736764c.png)

Reply:
get it